packages/remediations/src/steps/reviewSystems.js

```diff
@@ -0,0 +1,166 @@
+                                    ...config,
+                                    hasItems: true
+                                });
+                                const data = await inventoryApi?.mapTags(invData);
+                                return data;
+                            }}
+                            onLoad={({ mergeWithEntities, api }) => {
+                                registry.register(mergeWithEntities(entitiesReducer(onSelect)));
+                                setInventoryApi(() => api);
+                            }
+                            }
+                            bulkSelect={{
+                                id: 'select-systems',
+                                count: selected.size,
+                                items: [{
+                                    title: 'Select none (0)',
+                                    onClick: () => onSelectRows(false)
+                                },
+                                {
+                                    ...loaded && rows && rows.length > 0 ? {
+                                        title: `Select page (${ rows.length })`,
+                                        onClick: () => {
+                                            onSelectRows(true);
+                                        }
+                                    } : {}
+                                }],
+                                checked: selected.size > 0,
+                                onSelect: (value) => {
+                                    onSelectRows(value);
+                                }
+                            }}
+                            tableProps={{
+                                canSelectAll: false
+                            }}
+                        >
+                        </InventoryTable>
+                    </Router>
+                }
+            </StackItem>
+            { error && loaded &&
+                <StackItem>
+                    <ExclamationCircleIcon className="ins-c-remediations-error pf-u-mr-sm"/>
+                    <span className="ins-c-remediations-error">{error}</span>
+                </StackItem>
+            }
+        </Stack >
+    );
+};
+
+ReviewSystems.propTypes = {
+    systems: propTypes.arrayOf(propTypes.string).isRequired,
+    issues: propTypes.arrayOf(propTypes.shape({
+        description: propTypes.string,
+        id: propTypes.string
+    })).isRequired,
+    registry: propTypes.object.isRequired
+};
+
+const ReviewSystemsWithContext = (props) => {
+    const [ registry, setRegistry ] = useState();
+
+    useEffect(() => {
+        setRegistry(() => new ReducerRegistry({}, [ promiseMiddleware ]));
+    }, []);
+
+    return (
+        <Fragment>
+            {
+                registry && registry?.store  && <Provider store={registry?.store}>
+                    <ReviewSystems {...props} registry={registry} />
+                </Provider>
+            }
+        </Fragment>
+    );
```

Comment:
No need to wrap in fragment and extra check for `registry`
```suggestion
    return registry?.store  && <Provider store={registry.store}>
      <ReviewSystems {...props} registry={registry} />
    </Provider>;
```